#!/usr/bin/env python

PACKAGE = "rasberry_han"
from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("danger_region_radius", double_t, 0 , "Region Radius",1.0, 0.1,20)
gen.add("warning_region_radius", double_t, 0 , "Region Radius",2.0, 0.1,20)
gen.add("angle_fov", double_t, 0 , "Positive Angle Field of View",0.2, 0.1,1.0)
gen.add("timeout", double_t, 0 , "Timeout",1.0, 0.1,20)

exit(gen.generate(PACKAGE, "rasberry_han", "HumanProximityMonitor"))
